api/pkg/errors/errors.go

```diff
@@ -20,3 +21,13 @@ func NewInvalidInputError(reason string) error {
 func NewInvalidInputErrorf(reasonFormat string, a ...interface{}) error {
 	return fmt.Errorf("%w: %s", InvalidInputError, fmt.Sprintf(reasonFormat, a...))
 }
+
+// NewDeadlineExceededError create new DeadlineExceededError with specified reason
+func NewDeadlineExceededError(reason string) error {
+	return fmt.Errorf("%w: %s", DeadlineExceededError, reason)
+}
+
+// NewDeadlineExceededErrorf reate new DeadlineExceededError with specified reason string format
```

Comment:
nit: typo, should be `create new DeadlineExceededError with specified reason string format`